doc/Makefile.sphinx

```diff
@@ -61,23 +66,32 @@ check-sphinxbuild:
 	@echo "Confirming that sphinx-build is available..."
 	@if [ $(SPHINXTEST) -eq 1 ]; then echo $(SPHINXERROR); fi
 
-html: check-sphinxbuild source clean-build
+html-release: check-sphinxbuild source clean-build
 	@echo
 	@echo "Building HTML documentation"
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@chmod -R ugo+rX $(BUILDDIR)
+	mv $(BUILDDIR)/html .
```

Comment:
Should this rm -r html before doing the mv?  Or can we be guaranteed that something else has already done that?